src/main/java/edu/ksu/canvas/attendance/submitter/AssignmentValidator.java

```diff
@@ -102,8 +102,8 @@ public AttendanceAssignment validateConfigurationSetupExistence(AttendanceAssign
         try {
             assignmentOptional = canvasApiWrapperService.getSingleAssignment(courseId, oauthToken, attendanceAssignment.getCanvasAssignmentId().toString());
         } catch (IOException e) {
-            LOG.error("Error while getting assignment from canvas for section: " + attendanceAssignment.getAttendanceSection().getSectionId(), e);
-            throw new AttendanceAssignmentException(Error.NO_CONNECTION);
+            LOG.info("Error while getting assignment from canvas for section: " + attendanceAssignment.getAttendanceSection().getSectionId(), e);
```

Comment:
You're sending a weird mixed signal here. If this really is an error then it should be logged as an error and have a stacktrace. If it's not an error then it shouldn't have a stacktrace. Or this could possibly be logged at warning level.